aws_helper/config.go

```diff
@@ -22,5 +25,29 @@ func CreateAwsSession(awsRegion, awsProfile string) (*session.Session, error) {
 		return nil, errors.WithStackTraceAndPrefix(err, "Error finding AWS credentials (did you set the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables?)")
 	}
 
+	if awsRoleArn != "" {
+		err = assumeRole(session, awsRoleArn)
```

Comment:
BTW, `session` is a bad choice for a variable name here, as it shadows the `session` package that is imported. That has nothing to do with your code, just a screw up on my behalf. Could you rename it to something like `sess` instead to avoid confusion?